docs/README-AWS.md

```diff
@@ -64,57 +85,153 @@ Resources:
     Type: AWS::Serverless::Application
     Properties:
       Location:
-        ApplicationId: 'arn:aws:serverlessrepo:%REGION%:%ELASTIC_ACCOUNT_ID%:applications/elastic-serverless-forwarder'
+        ApplicationId: 'arn:aws:serverlessrepo:%REGION%:267093732750:applications/elastic-serverless-forwarder'
```

Comment:
Is this intentional?

Reply:
yes, it's the AWS account where it will be published